src/main/java/org/broadinstitute/consent/http/ConsentApplication.java

```diff
@@ -28,13 +28,12 @@
 import org.broadinstitute.consent.http.authentication.DefaultAuthenticator;
 import org.broadinstitute.consent.http.authentication.OAuthAuthenticator;
 import org.broadinstitute.consent.http.authentication.OAuthCustomAuthFilter;
-import org.broadinstitute.consent.http.health.GCSHealthCheck;
+import org.broadinstitute.consent.http.health.*;
```

Comment:
We don't use star imports, please revert this change and import your health check directly.

Reply:
See https://blog.marcnuri.com/intellij-idea-how-to-disable-wildcard-imports or https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports for how to do this in intellij

Reply:
Fixed